rules/falco_rules.yaml

```diff
@@ -3077,4 +3077,16 @@
 # Application rules have moved to application_rules.yaml. Please look
 # there if you want to enable them by adding to
 # falco_rules.local.yaml.
+- list: docker_binaries
+  items: [dockerd, containerd-shim, "runc:[1:CHILD]"]
```

Comment:
```suggestion
  items: [dockerd, containerd-shim, "runc:[1:CHILD]", pause]
```

Reply:
During the time I tested the rule I've seen some noise generated by proc.name = pause which should be whitelisted.